src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -39,22 +43,97 @@ internal NativeIntegerTypeSymbol(NamedTypeSymbol underlyingType) : base(underlyi
 
         public override SpecialType SpecialType => _underlyingType.SpecialType;
 
-        public override IEnumerable<string> MemberNames => Array.Empty<string>();
+        public override IEnumerable<string> MemberNames => GetMembers().Select(m => m.Name);
 
         /// <summary>
-        /// There are no members on <see cref="System.IntPtr"/> that should be exposed directly on nint:
-        /// constructors for nint other than the default parameterless constructor are not supported;
-        /// operators are handled explicitly as built-in operators and conversions;
-        /// 0 should be used instead of Zero;
-        /// sizeof(nint) should be used instead of Size;
-        /// + and - should be used instead of Add() and Subtract();
-        /// ToInt32(), ToInt64(), ToPointer() should be used from System.IntPtr only;
-        /// overridden methods Equals(), GetHashCode(), and ToString() are referenced from System.Object.
-        /// The one remaining member is <see cref="System.IntPtr.ToString(string)"/> which we could expose if needed.
+        /// Certain members from the underlying types are not exposed from the native integer types:
+        ///   constructors other than the default parameterless constructor are not supported;
+        ///   operators are handled explicitly as built-in operators and conversions;
+        ///   0 should be used instead of Zero;
+        ///   sizeof() should be used instead of Size;
+        ///   + and - should be used instead of Add() and Subtract();
+        ///   ToInt32(), ToInt64(), ToPointer() should be used from underlying types only.
+        /// The remaining members are exposed on the native integer types with appropriate
+        /// substitution of underlying types in the signatures.
         /// </summary>
-        public override ImmutableArray<Symbol> GetMembers() => ImmutableArray<Symbol>.Empty;
+        public override ImmutableArray<Symbol> GetMembers()
+        {
+            if (_lazyMembers.IsDefault)
+            {
+                ImmutableInterlocked.InterlockedInitialize(ref _lazyMembers, makeMembers(_underlyingType.GetMembers()));
+            }
+            return _lazyMembers;
+
+            ImmutableArray<Symbol> makeMembers(ImmutableArray<Symbol> underlyingMembers)
```

Comment:
>makeMembers [](start = 35, length = 11)

It doesn't look like the doc comment accurately reflects what is happening here. Consider clearly stating the intent what members are we expecting to clone. Otherwise, it is hard to tell whether the implementation matches the intent.